packages/frontend/src/components/DashboardFilter/ActiveFilters/index.tsx

```diff
@@ -213,6 +225,20 @@ const ActiveFilters: FC<ActiveFiltersProps> = ({
 
     return (
         <>
+            {!isEditMode && (
+                <Tooltip label="Reset all filters">
+                    <Button
+                        size="xs"
+                        variant="default"
+                        color="gray"
+                        onClick={() => {
+                            resetDashboardFilters();
+                        }}
+                    >
+                        <MantineIcon icon={IconRotate2} />
+                    </Button>
```

Comment:
It would be nice if this button only appeared when there are "dirty filters" or was disabled.